src/app-config/service.ts

```diff
@@ -45,6 +54,20 @@ export default class AppService {
     });
 
     const url = new URL(this.config.api_host);
+
+    if (process.env.TEST === '1') {
```

Comment:
This should be read in from the config file and not like this. TEST is okay, but the others are too flaky. Should default to Production but allow overriding from the config file.

Reply:
updated environment to be loaded and a part of the config file, with default values to be either test (if test) or production. Allows for overriding from the config file.